contracts/F3Parameters.sol

```diff
@@ -67,6 +71,9 @@ contract F3Parameters is Ownable {
     uint128 constant MIN_ACTIVATION_HEADROOM = 4 days;
     /// @dev The minimum headroom time in blocks.
     uint128 public constant MIN_ACTIVATION_HEADROOM_BLOCKS = MIN_ACTIVATION_HEADROOM / BLOCK_TIME;
+	
+	/// @dev The finality of the chain
+	uint64 public constant FINALITY = 900;
```

Comment:
```suggestion
	/// @dev The number of epochs for the chain to reach finality when using Expected Consensus
	uint64 public constant FINALITY_BLOCKS= 900;
```

Nit to make this self documenting?

Reply:
These changes will require re-deployment, do we think it is worth it?

Reply:
Oh yeah - good point.  Is redeployment also required even for adding more comments?

Reply:
Basically I'm wondering as a compromise if we make a comment change (maybe say this ideally should be called FINALITY_BLOCKS) and update the tests but leave the contract alone otherwise.

Reply:
The contract metadata won't match the repo code, but at least the code and ABI will match.